pgserver/logrepl/subscription.go

```diff
@@ -135,3 +140,16 @@ func SelectSubscriptionLsn(ctx *sql.Context, subscription string) (pglogrepl.LSN
 
 	return pglogrepl.ParseLSN(lsn)
 }
+
+func CommitAndUpdate(sqlCtx *sql.Context) error {
+	if err := adapter.CommitAndCloseTxn(sqlCtx); err != nil {
+		return err
+	}
+
+	err := UpdateSubscriptions(sqlCtx)
```

Comment:
Should this function be `CommitSubscriptionUpdates`? I.e., do `updateSubscriptions` first, and then `CommitAndCloseTxn`? Otherwise, this commits the previous transcation and starts a new one. Is this intended?

Reply:
This function seems confused here. Actually, it commit the latest transaction, and then update the `subscriptionMap` in memory.
I've removed this function in latest commit.